src/System.Collections.Specialized/src/System.Collections.Specialized.builds

```diff
@@ -4,8 +4,14 @@
   <ItemGroup>
     <Project Include="System.Collections.Specialized.csproj" />
     <Project Include="System.Collections.Specialized.csproj">
+      <TargetGroup>netstandard1.3</TargetGroup>
+    </Project>
+    <Project Include="System.Collections.Specialized.csproj">
       <TargetGroup>net46</TargetGroup>
```

Comment:
I was told by @joperezr we had to keep these, at least the netstandard1.3 and at least until we merged with master.  That's not the case?  I'd happily see all of this complication go away.